src/table/table.scss

```diff
@@ -155,6 +156,12 @@
   }
 }
 
+@mixin iui-table-header-actions-container {
```

Comment:
Does this mean the filter icon will be shown all the time instead of just on hover?

Reply:
no, it was a bug :D 
fixed